lib/src/console/console.dart

```diff
@@ -4,16 +4,20 @@ import 'dart:io';
 /// Allows to print log messages and error messages.
 class Console {
   /// Creates an instance with
-  const Console(this._output, this._error);
+  const Console(this._output, this._error, this._supportsAnsiEscapes);
```

Comment:
`Console` is exported by Cider, hence it is a part of external API (i.e. contract). Let's make the third argument optional, otherwise this technically makes a BC-breaking change.

Reply:
Actually disregard the first comment. Why do we need this argument at all, it should be possible to read it from `this.stdout`, shouldn't it?

Reply:
I didn't think it was part of the API.
This field is needed for formatted output to the console.

Reply:
See the comment below.